src/module.cc

```diff
@@ -1,5 +1,6 @@
-#include "node_report.h"
+// Copyright 2017 Nodereport
```

Comment:
I think there was also some discussion about removing the `All rights reserved` from that.